submissions/DmytryjK/HTML-CSS-Popup/css/components/header.css

```diff
@@ -0,0 +1,7 @@
+.header__logo-img {
```

Comment:
Why do you have styles for header in the next file if this file with header styles?

Reply:
Initially, I broke the styles into modules, a separate _header_, _fonts_, _variables_. I used to gulp for compile all modules to 1 main style.css file. Apparently somesing went wrong, when I uploaded them to the server.
Should I re-upload the sources, or just leave 1 main file style.css, and delete the rest?

Reply:
Better upload all modules without compiled style.css.

Reply:
> When you check all your files and you think your PR ready for review click re-request review ![image](https://user-images.githubusercontent.com/16196199/182035056-491e980e-6a61-49b9-aea7-ce98e2f3b94d.png)

I hope i fixed all problems for my project. But not enough understend problem on this link  - https://github.com/kottans/frontend-2022-homeworks/pull/37/files#diff-cb4e305b59c230dd9f2eb93110cd020f89c282589d220f4ea44f19689ab2f796
What i have to do?

Reply:
You did everything right. Your PR should contain only source files, no need to include compiled files to your PR.